.github/workflows/pkgci.yml

```
@@ -53,11 +53,12 @@ jobs:
   # Jobs that run unit tests on special hardware platforms or accelerators
   #############################################################################
 
-  test_amd_mi250:
-    name: Test AMD MI250
-    needs: [setup, build_packages]
-    if: contains(fromJson(needs.setup.outputs.enabled-jobs), 'test_amd_mi250')
-    uses: ./.github/workflows/pkgci_test_amd_mi250.yml
+  # Temporarily disbling MI250 runs due to downed Cirruscale GPU 945D machine
```

nit: The code in IREE doesn't explicitly say "Cirrascale" and all that the infrastructure really care about is the specified label, currently `nodai-amdgpu-mi250-x86-64`: https://github.com/iree-org/iree/blob/c3209357f059758f26686c67cfd180f800a71112/.github/workflows/pkgci_test_amd_mi250.yml#L20-L22

How about something more like this?

```suggestion
  # Disabled while no `nodai-amdgpu-mi250-x86-64` runners are online.
```

---

updated comment in the new commit